python/pyspark/sql/connect/README.md

```diff
@@ -49,3 +49,8 @@ To use the release version of Spark Connect:
 ./python/run-tests --testnames 'pyspark.sql.tests.connect.test_connect_basic'
 ```
 
+## Generate proto generated files for the Python client
+1. Install `buf`: https://docs.buf.build/installation
+2. Run `pip install grpcio==1.48.1 protobuf==4.21.6 mypy-protobuf==3.3.0`
+3. Run `./dev/generate_protos.sh`
```

Comment:
oops, should be `./connector/connect/dev/generate_protos.sh`

Reply:
@amaliujia

Reply:
I fix it in https://github.com/apache/spark/pull/38339

Reply:
oops I copied this from my local command which is not under the spark repo root directory. 

@zhengruifeng maybe you BTW update it in the PR that you pin dependency versions.